static/app/views/performance/mobile/components/deviceClassSelector.tsx

```diff
@@ -32,6 +41,17 @@ export function DeviceClassSelector({size = 'xs', clearSpansTableCursor}: Props)
       value={value}
       options={options ?? []}
       onChange={newValue => {
+        if (moduleName === ModuleName.APP_START) {
+          trackAnalytics('insight.app_start.spans.filter_by_device_class', {
+            organization,
+            filter: newValue.value,
+          });
+        } else if (moduleName === ModuleName.SCREEN_LOAD) {
+          trackAnalytics('insight.screen_load.spans.filter_by_device_class', {
+            organization,
+            filter: newValue.value,
+          });
+        }
```

Comment:
Could we dynamically set this?
Ie `insight.${moduleName}...`

Reply:
moduleName can also be `mobile-ui` which we do not track, so I explicitly included these cases